From b7ae59f661df5911ffa9cff1d5035b3e493fe730 Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@debian.org>
Date: Wed, 29 Nov 2017 14:31:10 -0800
Subject: [PATCH] ltsp-update-image: Exclude directories used by etckeeper to
 store /etc in revision control, as they may contain sensitive information.

---
 server/configs/ltsp-update-image.excludes | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/server/configs/ltsp-update-image.excludes b/server/configs/ltsp-update-image.excludes
index 93c93904..1453cd46 100644
--- a/server/configs/ltsp-update-image.excludes
+++ b/server/configs/ltsp-update-image.excludes
@@ -3,6 +3,11 @@ etc/epoptes/server.key
 etc/mysql/debian.cnf
 etc/NetworkManager/system-connections/*
 etc/udev/rules.d/??-persistent-*.rules
+etc/.bzr/
+etc/.etckeeper
+etc/.git/
+etc/.hg/
+etc/_darcs
 home/*
 lost+found/*
 media/*
-- 
2.11.0

